R/shared-methods.R

```diff
@@ -97,9 +97,8 @@ initialise_dataclass <- function(self, level = "1", regions,
   }
 
   if (!is.null(self$level_data_urls[[self$level]])) {
```

Comment:
Hmm this won't deal with overwriting shared levels and I am surprised unlist(list()) is better than c() here? 

Could add another if else to check for common_data_urls and only merge if both present?

Reply:
What do you think of the solution @RichardMN used in #250?

Reply:
I think its probably the best option?

Reply:
Ideally we PR into dev and then update all the PRs using it. Otherwise it is going to be a nightmare.

Reply:
#257